src/Tizen.Applications.Common/Tizen.Applications/ApplicationManager.cs

```diff
@@ -559,6 +559,61 @@ public static IEnumerable<RecentApplicationInfo> GetRecentApplications()
 
             return result;
         }
+
+        /// <summary>
+        /// Attaches the window of the child application to the window of the parent application.
+        /// </summary>
+        /// <remarks>
+        /// The application has to be signed with platform certificate.
+        /// </remarks>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static void AttachWindow(string parentAppId, string childAppId)
+        {
+            Interop.ApplicationManager.ErrorCode err = Interop.ApplicationManager.ErrorCode.None;
+
+            err = Interop.ApplicationManager.AppManagerAttachWindow(parentAppId, childAppId);
+            if (err != Interop.ApplicationManager.ErrorCode.None)
+            {
+                switch (err)
+                {
+                    case Interop.ApplicationManager.ErrorCode.InvalidParameter:
+                        throw new ArgumentException("Invalid argument.");
+                    case Interop.ApplicationManager.ErrorCode.IoError:
```

Comment:
I think we have to change native API implemention.
The function has to return the permission denied error.

Reply:
add exceptions for new errors in native api.